lib/stat_tracker.rb

```diff
@@ -184,9 +184,21 @@ def highest_scoring_visitor
         end
     end
     
-    # def highest_scoring_home_team
+    def highest_scoring_home_team
+        team_home_goals = Hash.new(0)
+        home_games_played = Hash.new(0)
+        @game_stats_data.each do |game_id, game_object|
+            team_home_goals[game_object.home_team_id] += game_object.home_goals
+            home_games_played[game_object.home_team_id] += 1    
+        end
 
-    # end
+        highest_home_team = team_home_goals.max_by do |team, goals|
+            goals.to_f / home_games_played[team].to_f
+       end
```

Comment:
Hey Shawn, this code looks good and it will work, but can you utilize the helper methods built for these cases